File: docs/components_page/components/nav/link_based.py
import dash_bootstrap_components as dbc

nav = dbc.Nav(
    [
        dbc.NavLink("Active", active=True, href="#"),
        dbc.NavLink("A link", href="#"),
        dbc.NavLink("Another link", href="#"),
        dbc.NavLink("Disabled", disabled=True, href="#"),
    ]
)

File: components_page/components/nav/link_based.R
library(dashBootstrapComponents)

nav <- dbcNav(
  list(
    dbcNavLink("Active", active = TRUE, href = "#"),
    dbcNavLink("A link", href = "#"),
    dbcNavLink("Another link", href = "#"),
    dbcNavLink("Disabled", disabled = TRUE, href = "#")
  )
)
